tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -381,7 +387,7 @@ namespace vz_example_viewer {
             } else {
               const diff = saliencyTotals[b.name] - saliencyTotals[a.name];
               if (diff != 0) {
```

Comment:
can diff be larger than 1, causing images to be below some features?
what happens when diff == 0 the function doesn't seem to return anything.

Reply:
sort function just needs to return pos value for two items that are equal position, negative value if it should be before the second item, and positive value if it should be after the second item. the magnitude of the number doesn't actually matter, just the sign. so this should do the right thing and keep images at top and beyond that, put features in sort order correctly.

Reply:
nevermind, it actually returns string sort if diff==0 :)